src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -255,6 +255,7 @@ static bool queryForward
   //
   std::string     out;
   int             r;
+  std::string     url;
```

Comment:
It seems this variable is not initialized...

Looking to similar code in senderThread.cpp I think this could be used:

```
url = ip + ":" + port + resource;
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3749/commits/d4e75f554d1327a17ec1235529378567fd5c4454